### PR TITLE
166572668 update admin interface tab

### PR DESCRIPTION
**Pull request by rckopteri**
#Fixed
* admin interface tab: Removed duplicate row
   
# Changed
* admin interface tab: checkboxes to radiobuttons
* transit visualization page - package info box - gray950 alert icon to red

### CODE REVIEW

File: ote/src/cljs/ote/app/controller/admin.cljs [hunk 1]
```diff
@@ -280,6 +281,35 @@
                :loading? false
                :results response))
 
+  UpdateInterfaceRadioFilter
+  (process-event [{f :radio-filter} app]
```
Comment (philosotractor) [inline ote/src/cljs/ote/app/controller/admin.cljs]: When each `cond` case evaluates the same symbol, `condp` simplifies the implementation:
```suggestion
  (process-event [{f :radio-filter} app]
   (condp = f
     :all
     (-> app
         (assoc-in [:admin :interface-list :filters :import-error] false)
         (assoc-in [:admin :interface-list :filters :db-error] false)
         (assoc-in [:admin :interface-list :filters :no-interface] false))
     :no-interface
     (-> app
         (assoc-in [:admin :interface-list :filters :import-error] false)
         (assoc-in [:admin :interface-list :filters :db-error] false)
         (assoc-in [:admin :interface-list :filters :no-interface] true))
     :db-error
     (-> app
         (assoc-in [:admin :interface-list :filters :import-error] false)
         (assoc-in [:admin :interface-list :filters :db-error] true)
         (assoc-in [:admin :interface-list :filters :no-interface] false))
     :import-error
     (-> app
         (assoc-in [:admin :interface-list :filters :import-error] true)
         (assoc-in [:admin :interface-list :filters :db-error] false)
         (assoc-in [:admin :interface-list :filters :no-interface] false))
     ;; :else
     (-> app
         (assoc-in [:admin :interface-list :filters :import-error] false)
         (assoc-in [:admin :interface-list :filters :db-error] false)
         (assoc-in [:admin :interface-list :filters :no-interface] false)))
   )
```

File: ote/src/cljs/ote/app/controller/admin.cljs [hunk 2]
```diff
@@ -280,6 +281,35 @@
...
+    (let [app (cond (= f :all)
```
Comment (philosotractor) [inline ote/src/cljs/ote/app/controller/admin.cljs]: `let` not needed, unless debugging

File: ote/src/cljs/ote/views/admin/interfaces.cljs [hunk 3]
```diff
@@ -182,58 +183,84 @@
...
+           :update! #(e! (admin-controller/->UpdatePublishedFilter %))}
+          {:name :radio-group
+           :type :component
+           :width "100%"
+           :container-class "col-xs-12 col-sm-10 col-md-10"
+           :full-width? true
+           :component (fn [data]
+                        [:div {:style {:display "flex"
+                                       :flex-direction "row"}}
+                         (.log js/console "data " (pr-str data))
```
Comment (philosotractor) [inline ote/src/cljs/ote/views/admin/interfaces.cljs]: Debug log forgotten here?

File: ote/src/cljs/ote/app/controller/admin.cljs (hunk 1 above)
Comment (philosotractor) [reply ote/src/cljs/ote/app/controller/admin.cljs]: Or use `case` instead of the `condp` in the example I pasted :)

Comment (rckopteri) [reply ote/src/cljs/ote/app/controller/admin.cljs]: I went with case and update-in. Thanx!

File: ote/src/cljs/ote/app/controller/admin.cljs (hunk 2 above)
Comment (rckopteri) [reply ote/src/cljs/ote/app/controller/admin.cljs]: yup. I removed it.

File: ote/src/cljs/ote/views/admin/interfaces.cljs (hunk 3 above)
Comment (rckopteri) [reply ote/src/cljs/ote/views/admin/interfaces.cljs]: yes. thanks

**Comment by philosotractor** [comment]
❤️

**Pull request merged**